tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -53,21 +59,23 @@ virtual void TearDown() {
     CFRelease(propertyValue);
 }
 
-const NSString* c_familyName = @"Segoe UI";
+const NSString* c_familyName = @"Metadata Test";
 const NSString* c_subFamilyName = @"Regular";
 const NSString* c_styleName = @"Regular";
-const NSString* c_uniqueName = @"Segoe UI Regular";
-const NSString* c_fullName = @"Segoe UI";
+const NSString* c_uniqueName = @"Metadata Test Regular";
+const NSString* c_fullName = @"Metadata Test";
 
-const NSString* c_postscriptName = @"SegoeUI";
-const NSString* c_trademarkName = @"Segoe is a trademark of the Microsoft group of companies.";
+const NSString* c_postscriptName = @"MetadataTest-Regular";
+const NSString* c_trademarkName = @"MetadataTest is not a trademark.";
 const NSString* c_manufacturerName = @"Microsoft Corporation";
 const NSString* c_designerName = nullptr;
 const NSString* c_descriptionName = nullptr;
-const NSString* c_vendorURLName = @"http://www.microsoft.com/typography/fonts/";
+const NSString* c_vendorURLName = @"https://developer.microsoft.com/en-us/windows/bridges/ios";
 const NSString* c_designerURLName = nullptr;
-const NSString* c_licenseURLName = @"http://www.microsoft.com/typography/fonts/";
+const NSString* c_licenseURLName = @"https://developer.microsoft.com/en-us/windows/bridges/ios";
 const NSString* c_sampleTextName = nullptr;
+const NSString* c_copyrightName = @"Copyright (c) Microsoft";
+const NSString* c_versionName = @"Version 1.01 ";
```

Comment:
How strange that it leave a space at the end; I was even cautious to have it not do that.